src/views/Articles.jsx

```diff
@@ -0,0 +1,70 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import dateFn from 'date-fns';
+
+/**
+ * @description - returns all articles created by all authors
+ * @param {props} articles - all authors article
+ * @return {JSX} 
+ */
+class Articles extends Component {
+  /**
+ * @description time an article was created
+ * @param {object} article - each authors article
+ * @return {array} fiveStaAauhtors - author's rated highly
+ */
+  time(article) {
```

Comment:
#### Nitpick
- Should be isolated out of here.

Reply:
Done